RadixWallet/Features/CreateAccount/Children/NameAccount/NameAccount+View.swift

```diff
@@ -73,7 +73,7 @@ extension NameAccount {
 						)
 						.keyboardType(.asciiCapable)
 						.autocorrectionDisabled()
-						.padding(.bottom, .medium3)
+						.padding(.bottom, .large3)
```

Comment:
Updated to match [designs](https://zpl.io/Pq0p3ek) after removing vertical padding inside [ToggleView](https://github.com/radixdlt/babylon-wallet-ios/pull/1307/files#diff-2dda738b7b26f10c4d0462e34d1922d972f85aaf6e58dc2ab0019924e45488aeL34).